src/mpl/include/mpl_gpu.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (C) by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+#ifndef MPL_GPU_H_INCLUDED
+#define MPL_GPU_H_INCLUDED
+
+#include "mplconfig.h"
+
+#include "mpl_gpu_fallback.h"
+
+typedef enum {
+    MPL_GPU_POINTER_HOST = 0,
+    MPL_GPU_POINTER_REGISTERED,
```

Comment:
Can you change these to `REGISTERED_HOST` and `UNREGISTERED_HOST`?  `REGISTERED` is host too, right?

Reply:
fixed.